<launch>

<include file="$(find microstrain_inertial_driver)/launch/microstrain.launch">
    <arg name = "node_name" value="footSensor" />
    <arg name = "port" value="/dev/ttyACM0" />
    <arg name = "namespace" value="footSensor" />
</include>

<include file="$(find microstrain_inertial_driver)/launch/microstrain.launch">
    <arg name = "node_name" value="shankSensor" />
    <arg name = "port" value="/dev/ttyACM1" />
    <arg name = "namespace" value="shankSensor" />
</include>


<node name="infer_speed" pkg="v_estimate" type="infer_speed.py" output="screen">
</node>

<node name="filter_speed" pkg="v_estimate" type="filter_speed.py" output="screen">
</node>

<node name="logger" pkg="v_estimate" type="logger.py" output="screen">
</node>

<node name="gps" pkg="v_estimate" type="gps_publisher.py" output="screen">
</node>


<node name="lcd" pkg="v_estimate" type="lcd_display.py" output="screen">
</node>
 

</launch>